packages/core/src/atom.ts

```diff
@@ -200,9 +201,9 @@ export const createCtx = ({
   callLateEffect = callSafely,
   callNearEffect = callSafely,
 }: CtxOptions = {}): Ctx => {
-  let caches = new WeakMap<AtomProto, AtomCache>()
-  let read = (proto: AtomProto): undefined | AtomCache => caches.get(proto)
-  let logsListeners = new Set<Fn<[Logs, Error?]>>()
+  const caches = new WeakMap<AtomProto, AtomCache>()
```

Comment:
What does this for? Move it to separate PR pls

Reply:
Sorry, don't want to move it to PR. Just find out what variables are mutatable.